README.md

```diff
@@ -18,28 +18,126 @@ More complete docs can be found at [pkg.go.dev](https://pkg.go.dev/github.com/op
 
 ## Usage Example
 
+```go
```

Comment:
Note: this example previously did not have correct formatting and did not handle the errors coming from recording values, I changed it to add those even though it's unrelated to the rest of the work in this PR.